### PR TITLE
New FQA How to change my avatar

### PR DESCRIPTION
**Pull request by JilSoete**
I added a new FQA: How do I change my avatar?

### CODE REVIEW

**Comment by sylee-** [comment]
For consistency with the rest of the FAQs that we are changing, I recommend the following:

> Navigate to the picture or video you want to delete (make sure you're looking at the enlarged picture or playable video). Below the image on the lefthand side, click on "options'' to open a small dropdown menu.  One of the three options is:

> _Set as Avatar_
This will set the image as your main Avatar for your profile so you can show yourself off!

_______________

We have a more comprehensive answer at https://github.com/fetlife/help/pull/96, but I can see how having a separate answer for this is a good idea.

File: 02_profiles/02_how-do-i-change-my-avatar.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# How do I change my avatar?
```
Comment (nielsenramon) [inline 02_profiles/02_how-do-i-change-my-avatar.md:1]: `How do I update my avatar?`

File: 02_profiles/02_how-do-i-change-my-avatar.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+To update your avatar:
+
+ 1. Go to your profile.
```
Comment (nielsenramon) [inline 02_profiles/02_how-do-i-change-my-avatar.md:5]: We don't use `.` at end of sentences :p

File: 02_profiles/02_how-do-i-update-my-avatar.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# How do I update my avatar?
+
+You can update your avatar at any time by:
+
+1. Visiting the picture you wish to use
+2. Selecting "options" below the picture
+3. Selecting "Set as Avatar"
+
+Alternatively, you can directly upload a new picture and set it as your avatar here: https://fetlife.com/pictures/new
```
Comment (nielsenramon) [inline 02_profiles/02_how-do-i-update-my-avatar.md:9]: Remove `directly` here please.

File: 02_profiles/02_how-do-i-update-my-avatar.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# How do I update my avatar?
+
+You can update your avatar at any time by:
+
+1. Visiting the picture you wish to use
```
Comment (nielsenramon) [inline 02_profiles/02_how-do-i-update-my-avatar.md:5]: `... you wish to use as your avatar`

**Pull request merged**